lib/src/klio/transforms/core.py

```diff
@@ -37,6 +37,10 @@ class RunConfig(object):
     _lock = threading.Lock()
     _config = None
 
+    @classmethod
```

Comment:
Curious, why is this a class method? And not an attribute of the class (like `_lock` or `_config`)? Every time `_logger()` is called, it's a function call (`getLogger(...)`) when it could just be called once.

Reply:
yeah not sure why I did that.  Fixed it!